### PR TITLE
Support loading prompts from yaml

### PR DESCRIPTION
**Pull request by zewelor**
Added support to load prompt from yaml file

### CODE REVIEW

File: spec/fixtures/prompt/prompt_template.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+_type: prompt
+input_variables:
+  - adjective
+  - content
+template: Tell me a {adjective} joke about {content}.
```
Comment (andreibondarev) [inline spec/fixtures/prompt/prompt_template.yaml:5]: Does yaml support multi-line strings?

Comment (andreibondarev) [reply spec/fixtures/prompt/prompt_template.yaml:5]: That was my complaint with JSON -- that you can't do multiline strings and a lengthy template would just be a single line with tons of line break characters.

**Comment by zewelor** [comment]
Yes in YAML its quite easy to do multiline string. Changes spec fixture to show an example. Here is some site to take a look at different options how multiline strings can be used in yaml: https://yaml-multiline.info/

**Comment by alchaplinsky** [comment]
Yeah, I think `yaml` is better format for prompts than JSON. Prompts will be more readable.

**Comment by andreibondarev** [comment]
@zewelor This is great, thank you!

**Pull request merged**